ui/app/routes/jobs/index.js

```diff
@@ -32,7 +32,7 @@ export default class IndexRoute extends Route.extend(
     controller.set('namespacesWatch', this.watchNamespaces.perform());
     controller.set(
       'modelWatch',
-      this.watchJobs.perform({ namespace: controller.qpNamesapce })
+      this.watchJobs.perform({ namespace: controller.qpNamesapce, meta: true })
```

Comment:
`unrelated` `nit`: Do you see that typo to the left of your change?

Reply:
`unrelated` `question`: We don't poll `/jobs` though? I was thinking about removing this. Any concerns you can think of while we're here?

Reply:
We do poll existing jobs for changes; if you are on the index page and a `pending` job changes to `ready`, for example, that update will propagate to the list.